docs/project/reward-algorithm_v2.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: reward-algorithm_v2
+title: Reward Algorithm_v2
+sidebar_label: Reward Algorithm_v2
+slug: /reward-algorithm_v2
+---
+
+This section describes how we are palanning to reward the community. 
+
+Our **Reward Algorithm** is a unique, fair and transparent algorithm that will handle how rewards are being distributed to the network members.
+## Rewards Allocation Methodology
+
+The rewards allocated for a cell will be distributed to the first N devices with best station reward score, where N=capacity of cell. In case two devices share the same score then they will be sorted based on seniority and the more senior one will get the rewards.
+
+ 
+
+
+:::note For example:
+If the capacity of a cell is 5 but has 6 stations and the Max Cell Reward is 20 WXM then the first 5 devices based on their performance in the previous day `station_reward_score` will be rewarded with a `station_reward_score`*`20/5`. This would have also been the case, if the specific cell had fewer than 5 stations.
+:::
+To read more please see our <a href="https://docs.google.com/document/d/1_z-sdCoSpm7H5kH0Ko8grUn7IatmAObSdkQULr21YGc/edit#heading=h.jgrgzl8vvkoy">whitepaper</a> 

```

Comment:
We should **not** use the google docs link at any cases, as that might change in the future. Please let's use the static one we have control of:
[https://weatherxm.com/whitepaper](https://weatherxm.com/whitepaper)

Reply:
you are correct. I only linked to the google doc, because it could go straight to the specific section.fixed